pajbot/web/utils.py

```diff
@@ -70,7 +71,9 @@ def download_logo(client_id, streamer):
 
     twitchapi = TwitchAPI(client_id)
     try:
-        data = twitchapi.get(["users", streamer], base="http://127.0.0.1:7221/kraken/")
+        data = twitchapi.get(
+            ["users", streamer], base=f"http://{os.environ.get('KRAKEN_URL', '127.0.0.1:7221')}/kraken/"
```

Comment:
Also in line with the `[redis]` configuration thing, if the redis config thing is possible, then it would be nice to configure this in the config as well (maybe under [twitchapi] and [webtwitchapi]). When the api proxy was introduced this was a quick fix because the (back then) deadline for v3 was coming soon.